script/validate.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+git diff --exit-code --quiet --name-only -- "*.scala" "*.sbt" "script/validate.sh" ".travis.yml" ".scalafmt.conf"
```

Comment:
This PR should run sbt since `script/validate.sh` is included.

Maybe ...

```suggestion
git diff ${TRAVIS_COMMIT_RANGE} --exit-code --quiet --name-only -- "*.scala" "*.sbt" "script/validate.sh" ".travis.yml" ".scalafmt.conf"
```